notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,457 @@
+    "\n",
+    "This notebook uses five pre-trained models, for the following styles: Mosaic, Rain Princess, Candy, Udnie and Pointilism. The models are from the [ONNX Model Repository](https://github.com/onnx/models) and are based on the research paper [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/abs/1607.08022). Final part of this notebook shows live inference results from a webcam. Additionally, you can also upload a video file.\n",
+    "\n",
+    "> **NOTE**: To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server, the webcam will not work. However, you can still do inference on a video."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import collections\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "from IPython.display import Markdown as md\n",
+    "from IPython import display\n",
+    "from ipywidgets import interactive, ToggleButtons\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Select one of the styles: Mosaic, Rain Princess, Candy, Udnie and Pointilism."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Option to select different styles    \n",
+    "styleButtons = ToggleButtons(\n",
+    "    options=['MOSAIC', 'RAIN-PRINCESS', 'CANDY', 'UDNIE', 'POINTILISM'],\n",
+    "    description=\"Click on the Style:\",\n",
+    "    disabled=False,\n",
+    "    style={'description_width': '300px'})\n",
+    "   \n",
+    "interactive(lambda option: print(option), option=styleButtons)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Model\n",
+    "\n",
+    "### Download the Model\n",
+    "\n",
+    "Model for the selected style will be downloaded to `base_model_dir` if they have not been downloaded before. The selected model comes from the ONNX Model Zoo, which means it must be converted into OpenVINO Intermediate Representation (OpenVINO IR)."
```

Comment:
Add the link for ONNX Model Zoo. ONNX model can be loaded directly in OpenVINO, without the need to convert the model to IR. We need to mention that and give the users this option, it could be done using a commented cell after the IR conversion.

Reply:
Included the model repository link as well as the commented line to read ONNX model directly.